CONTRIBUTING.md

```diff
@@ -13,7 +13,11 @@ section.
 ## Modules
 
 * Prefer the use of the singular form in module name (or use "multi")
-* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module)
+* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module) but remove sections with no meaningful content.
```

Comment:
No, it was a live discussion in Odoo experience with the people of ACSONE. It was referring to this PR: https://github.com/OCA/bank-payment/pull/173. @phschmidt, can you add your point of view?